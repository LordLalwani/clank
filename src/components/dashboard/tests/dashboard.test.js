import { cleanup } from "@testing-library/react";
import React from "react";
import renderer from "react-test-renderer";
import Dashboard from "../components/dashboard";

afterEach(cleanup);

describe("render tests", () => {
  it("should render signIn component correctly", () => {
    const tree = renderer.create(<Dashboard />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
