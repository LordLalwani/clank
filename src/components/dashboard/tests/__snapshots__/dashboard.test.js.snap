// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render tests should render signIn component correctly 1`] = `
<div
  className="sc-bdnylx dxBFCn"
>
  dashboard
</div>
`;
